lib/ui/platform_dispatcher.dart

```diff
@@ -623,6 +623,16 @@ class PlatformDispatcher {
   @FfiNative<Void Function(Handle)>('PlatformConfigurationNativeApi::SetIsolateDebugName')
   external static void _setIsolateDebugName(String name);
 
+  /// Requests the Dart VM to adjusts the GC heuristics based on the requested `performance_mode`.
+  ///
+  /// See [DartPerformanceMode] for more information on individual performance modes.
+  void requestDartPerformanceMode(DartPerformanceMode mode) {
```

Comment:
Maybe add a note that this is a no-op on the web and requests to change the mode may be ignored or changed by the platform at any time.